src/chart/Summary.js

```diff
@@ -92,14 +100,11 @@
                 "min-height": this.minHeight_exists() ? this.minHeight() + "px" : null
             })
         ;
-        this._headerDiv
-            .style("color", this.colorStroke())
-            .text(row[1])
-        ;
-        this._textDiv
-            .text(row[0])
-        ;
+        
         var context = this;
+        this._headerDiv.style("color", this.colorStroke()).text(row[1]!== undefined ? context.fields()[1].transform(row[1]):"");
```

Comment:
@mounicarachakonda might want to break this on multiple lines d3 style to make it easier to understand whats going on there ... your setting the style, then setting the text of the d3 selected object